pages/index.js

```diff
@@ -124,37 +115,8 @@ const Home = (props) => {
         </div>
 
         <FadeInSection>
-          <div className={styles.statisticsWithIcon}>
```

Comment:
I think you accidentally removed this Statistics section. Can help to add it back?